src/main/java/ome/system/Principal.java

```diff
@@ -16,15 +15,15 @@
  * implementation of {@link java.security.Principal}. Specialized for Omero to
  * carry a {@link ExperimenterGroup group}, an {@link EventType event type} and
  * a {@link Permissions umask}.
- * 
+ *
  * @author Josh Moore, josh.moore at gmx.de
  * @version $Revision$, $Date$
  * @see EventType
  * @see ExperimenterGroup
  * @see Permissions
  * @since 3.0
  */
-public class Principal implements IPrincipal, Serializable {
+public class Principal implements Serializable {
```

Comment:
This does not match what is currently on develop ``implements java.security.Principal, Serializable``
why is that dropped?